backend/docker-compose.yml

```diff
@@ -6,35 +6,38 @@ services:
       context: .
       target: demo
       args:
-        SECRET_KEY: ${SECRET_KEY:?}
+        SECRET_KEY: tmp_key
     image: spellbook-backend
-    expose:
-      - 8000
+    ports:
+      - 8000:80
     depends_on:
+      db:
+        condition: service_healthy
+    links:
       - db
     volumes:
       - static_volume:/home/app/web/staticfiles:rw
     environment:
       SQL_ENGINE: django.db.backends.postgresql
-      SQL_DATABASE: spellbook_db_prod
-      SQL_USER: ${DB_USER:?}
-      SQL_PASSWORD: ${DB_PASSWORD:?}
+      SQL_DATABASE: spellbook_db_test
+      SQL_USER: test_user
+      SQL_PASSWORD: test_password
       SQL_HOST: db
       SQL_PORT: 5432
       DATABASE: postgres
     restart: always
-  nginx:
```

Comment:
Why the need of altering nginx here?

Reply:
Putting it directly on the image for the api makes our lives a lot easier with kubernetes since it means that every pod will only need to run one image.